src/dmqnode/storage/engine/DiskOverflow.d

```diff
@@ -754,7 +755,7 @@ class DiskOverflow: DiskOverflowInfo
         off_t pos = this.readHeader(header, channel.first_offset);
```

Comment:
There's still an in contract above (line 734).

Reply:
This is one of the contracts and assertions that I left because they are relevant for debugging only.